cypress/integration/models/developer/applicationinventory/applicationinventory.ts

```diff
@@ -361,7 +362,8 @@ export class ApplicationInventory {
             ApplicationInventory.clickApplicationInventory();
             selectItemsPerPage(100);
             this.selectApplication();
-            clickByText(button, assess);
+            cy.waitForReact();
+            cy.react("p", reactSelectorAssessButton).click();
```

Comment:
clickByText(button, assess); is working now 
![Uploading Screenshot from 2022-07-15 11-49-42.png…]()